### PR TITLE
Ret 1472

### PR DESCRIPTION
**Pull request by olyode**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RET-1472

### Change description ###
Added PCQ configurations to enable PCQ service invocation

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X ] No
```

### CODE REVIEW

File: definitions/json/AuthorisationCaseField.json [hunk 1]
```diff
@@ -1,4 +1,34 @@
 [
```
Comment (marcjudge) [inline definitions/json/AuthorisationCaseField.json:1]: Large volume of changes in this file doesn't look right

File: yarn.lock [hunk 2]
```diff
@@ -3,102 +3,102 @@
```
Comment (marcjudge) [inline yarn.lock:4]: Please undo changes here so we're back using https://registry.yarnpkg.com

**Review by marcjudge — CHANGES_REQUESTED** [review]
What is that ecm-docker file you've added?

File: definitions/json/AuthorisationCaseField.json (hunk 1 above)
Comment (olyode) [reply definitions/json/AuthorisationCaseField.json:1]: Surprised when I merged master also but fixed now

**Review by pmumbere — APPROVED** [review]

File: definitions/json/AuthorisationCaseField.json [hunk 3]
```diff
@@ -1,4 +1,34 @@
...
+  },
+  {
+    "CaseTypeId": "ET_EnglandWales",
+    "CaseFieldID": "ClaimantPcqId",	
+    "UserRole": "caseworker-employment",
+    "CRUD": "R"
+  },
+  {
+    "CaseTypeId": "ET_EnglandWales",
+    "CaseFieldID": "pcqId",
```
Comment (priyankaVerma21) [inline definitions/json/AuthorisationCaseField.json:28]: "CaseFieldID": "pcqId",	 caseFieldId is referencing pcqId but it is not referencing in caseField.json. Is this a mistake.

File: definitions/json/AuthorisationCaseField.json [hunk 4]
```diff
@@ -1,4 +1,34 @@
 [
+  {
+    "CaseTypeId": "ET_EnglandWales",
+    "CaseFieldID": "scannedDocuments",	
+    "UserRole": "caseworker-ET-pcqextractor",
```
Comment (priyankaVerma21) [inline definitions/json/AuthorisationCaseField.json:5]: This role (caseworker-ET-pcqextractor)  is only needed if we need to consolidate Pcq Id . If you are sending ccdCaseId all the time then consolidation is not needed and all the entries using this role can be removed.

File: yarn.lock [hunk 5]
```diff
@@ -1284,9 +1284,13 @@ yargs@16.2.0:
     require-directory "^2.1.1"
     string-width "^4.2.0"
     y18n "^5.0.5"
+<<<<<<< HEAD
```
Comment (marcjudge) [inline yarn.lock:1287]: Looks like a merge wasn't done correctly, please remove the text on this line and 1289

Comment (pmumbere) [reply yarn.lock]: Resolved. Not sure what git did.

**Review by marcjudge — APPROVED** [review]

**Pull request merged**